st2auth/st2auth/app.py

```diff
@@ -65,9 +65,9 @@ def setup_app(config={}):
 
     app = router.as_wsgi
 
+    app = ErrorHandlingMiddleware(app)
```

Comment:
Do you want me to add `# Ordering is important` or do you want me to explain why exactly one middleware should go before the other? Don't you think that adding such a detailed description in three different places would require us to constantly keep them up to date and will introduce even more confusion than simply reading the code?